src/__acceptance_tests__/__ssr_pages__/header.tsx

```diff
@@ -6,19 +6,27 @@ import {
     MainSectionHeader,
     ButtonPrimary,
     Stack,
+    Text3,
+    Text8,
+    skinVars,
 } from '../../..';
 
 const HeaderTest = (): JSX.Element => (
     <Stack space={8}>
         <HeaderLayout
-            header={
-                <Header
-                    title="La última factura de diciembre ya esta disponible"
-                    preamount="Cuota mensual (IVA incluido)"
-                    amount="60,44 €"
-                    button={<ButtonPrimary href="asdf">Descargar factura</ButtonPrimary>}
-                    subtitle="Y esto es un subtitulo"
-                />
+            header={<Header title="La última factura de diciembre ya esta disponible" />}
+            extra={
+                <Stack space={16}>
+                    <Stack space={8}>
+                        <Text3 regular color={skinVars.colors.textPrimary}>
+                            Cuota mensual (IVA incluido)
+                        </Text3>
+                        <Text8 color={skinVars.colors.textPrimary}>60,44 €</Text8>
+                    </Stack>
+
+                    <ButtonPrimary href="asdf">Descargar factura</ButtonPrimary>
+                    <Text3 regular>Y esto es un subtitulo</Text3>
```

Comment:
I think you could change this by a placeholder. We're testing the Header SSR, the extra content is unrelated to this test

Reply:
done